internal/dto/mailing/send.go

```diff
@@ -0,0 +1,7 @@
+package dto
+
+type MailingReq struct {
+	Emails          []string `json:"emails" validate:"required,min=1,dive,email"`
+	PasswordMessage string   `json:"passwordMessage" binding:"required"`
```

Comment:
Creo que sobraría message, ya sabemos que las variables serán utilizadas para el mensaje.
```suggestion
	Password string   `json:"password" binding:"required"`
```

Reply:
cambiado: https://github.com/OwlByTech/optitech-api/pull/3/commits/8f279e7ce2b4af6dbada59de0c05143176aa1ff1